### PR TITLE
Update 02-How to Estimate Programming Time.md

### PR DESCRIPTION
**Pull request by mmarica**
Fixed a typo

### CODE REVIEW

File: en/1-Beginner/Team-Skills/02-How to Estimate Programming Time.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ When not possible to take the time for some investigation, you should first esta
...
+Pad explicitly instead. If a task will probably take one day - but might take ten days if your approach doesn't work - note this somehow in the estimate if you can; if not, at least do an average weighted by your estimates of the probabilities. Any risk factor that you can identify and assign an estimate that should go into the schedule. One person is unlikely to be sick in any given week. But a large project with many engineers will have some sick time; likewise vacation time. And what is the probability of a mandatory company-wide training seminar? If it can be estimated, stick it in. There are of course, unknown unknowns, or *unk-unks*. Unk-unks by definition cannot be estimated individually. You can try to create a global line item for all unk-unks, or handle them in some other way that you communicate to your boss. You cannot, however, let your boss forget that they exist, and it is devilishly easy for an estimate to become a schedule without the unk-unks considered.
```
Comment (braydie) [inline en/1-Beginner/Team-Skills/02-How to Estimate Programming Time.md]: Hey, @mmarica, could you roll this change back to 'to' instead of 'that' as I don't think change clarifies the sentence. Perhaps "Any risk factor that you can identify and assign an estimate to, should go into the schedule" could work better?

**Comment by Idyllei** [comment]
How about rewording similar to the following: "Any risk factor which you can identify and to wish you can assign an estimate should be put into the schedule."

File: en/1-Beginner/Team-Skills/02-How to Estimate Programming Time.md (hunk 1 above)
Comment (mmarica) [inline en/1-Beginner/Team-Skills/02-How to Estimate Programming Time.md]: Hi, your version sounds much better :)
